qiskit/circuit/gate.py

```diff
@@ -235,7 +235,7 @@ def validate_parameter(self, parameter):
             warn("Gate param type %s is being deprecated as of 0.16.0, and will be removed "
                  "no earlier than 3 months after that release date. "
                  "Considering creating your own Gate subclass with the method validate_parameter "
-                 " to allow this param type." % type(parameter), DeprecationWarning, 3)
+                 "to allow this param type." % type(parameter), DeprecationWarning, 5)
```

Comment:
Is this actually correct? The current 3 doesn't seem to match anything correctly, but I guess this depends on how many subclasses there are I guess.

Reply:
It's also because this is run by the setter...